lib/cn/csched_sp3.c

```diff
@@ -1263,6 +1261,7 @@ sp3_trees_add(struct sp3 *sp)
         list_del(&spt->spt_tlink);
         list_add(&spt->spt_tlink, &sp->mon_tlist);
 
+        atomic_dec(&sp->sp_addprune_count);
```

Comment:
This dec was missing which caused every iteration of the monitor thread to scan all the trees to see if any needed to be pruned (and/or added).  Long standing bug, courtesy of yours truly, bah...